WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -176,12 +176,16 @@ private extension ProductVariationsViewController {
             return
         }
 
-        let moreButton = UIBarButtonItem(image: .moreImage,
-                                     style: .plain,
-                                     target: self,
-                                     action: #selector(presentMoreActionSheet(_:)))
+        var moreButton: UIBarButtonItem? = nil
+        // Do not display the "more" button with the bulk update option if we do not have any variations
+        if self.product.variations.isNotEmpty {
+            moreButton = UIBarButtonItem(image: .moreImage,
+                                         style: .plain,
+                                         target: self,
+                                         action: #selector(presentMoreActionSheet(_:)))
+        }
 
-        navigationItem.rightBarButtonItems = [moreButton]
+        navigationItem.rightBarButtonItem = moreButton
```

Comment:
Actually I think I need the `moreButton` to be either nil or the "more" button so that I can remove/add the button when I return from the add/remove variation flow. So if I have one variation and when I delete it I need the same view controller to remove the more.